cli/src/main.rs

```diff
@@ -6,7 +6,11 @@ use base64;
 use clap::{crate_name, crate_version, App, AppSettings, SubCommand};
 use serde_json::json;
 
-use anycloud::deploy::{client_error, get_config, info, new, terminate, upgrade, ALAN_VERSION};
+use anycloud::deploy::{
+  add_cred, add_deploy_config, client_error, edit_cred, edit_deploy_config, get_config, info,
+  list_creds, list_deploy_configs, new, remove_cred, remove_deploy_config, terminate, upgrade,
+  ALAN_VERSION,
+};
```

Comment:
Maybe it would be easier to read below if it was just `use anycloud::deploy;` here so you get `deploy::new(...` instead of just `new(...` (as an example).

It is more text to prefix `deploy::` everywhere, but it might be more legible.

Reply:
I like that